checkov/common/parallelizer/parallel_runner.py

```diff
@@ -72,12 +78,12 @@ def func_wrapper(original_func: Callable[[Any], _T], items_group: List[Any], con
                         result = original_func(*item)
                     else:
                         result = original_func(item)
-                except Exception:
+                except Exception as e:
                     logging.error(
                         f"Failed to invoke function {func.__code__.co_filename.replace('.py', '')}.{func.__name__} with {item}",
                         exc_info=True,
                     )
-                    result = None
+                    result = ParallelRunException(e)
```

Comment:
why did you change this? Does this change result in a different behaviour in our CLI runs? (for example now raising an exception instead of returning None)

Reply:
currently we are breaking our "contract" because this used to return None instead of failing like the other implementations of parallel run.
The PR checks are failing randomly on this issue and we currently have no visibility to what is causing the failure.

Reply:
Not sure I follow - what is the current situation and what has changed due to this fix?
I'm mainly trying to understand if users trying to run checkov might have "hidden" exceptions which will now be raised. This might be a good fix but something I want to understand the impact of before approving, as I fear even if this is a good fix a lot of our users will get unexpected errors now.

Reply:
The current situation is in this specific case the parallel execution will return None instead of raising an error. The other implementations will raise an error.